source/transfers/test/TestTransferHandlerRegistry-unit.js

```diff
@@ -0,0 +1,68 @@
+const TransferHandlerRegistry = artifacts.require('TransferHandlerRegistry')
+const { takeSnapshot, revertToSnapshot } = require('@airswap/test-utils').time
+const { equal, emitted } = require('@airswap/test-utils').assert
+const { EMPTY_ADDRESS } = require('@airswap/order-utils').constants
+
+contract('TransferHandlerRegistry Unit Tests', async accounts => {
+  const erc20Asset = accounts[2]
+  let snapshotId
+  let transferhandlerregistry
+
+  beforeEach(async () => {
+    const snapShot = await takeSnapshot()
+    snapshotId = snapShot['result']
+  })
+
+  afterEach(async () => {
+    await revertToSnapshot(snapshotId)
+  })
+
+  before('Deploy TransferHandlerRegistry', async () => {
+    transferhandlerregistry = await TransferHandlerRegistry.new()
+  })
+
+  describe('Test fetching non-existent handler', async () => {
+    it('test fetching non-existent handler, returns null address', async () => {
+      equal(
+        EMPTY_ADDRESS,
+        await transferhandlerregistry.getTransferHandler.call('0x80ac58cd'),
+        'Returns actual non-zero address'
+      )
+    })
+  })
+
+  describe('Test adding to handler', async () => {
+    it('test adding, should pass', async () => {
+      await emitted(
+        await transferhandlerregistry.addTransferHandler(
+          '0x80ac58cd',
```

Comment:
can we turn `0x80ac58cd` into a constant at the top and name it with a self descriptive variable name?

Reply:
yup, it didn't exist earlier.